docs/articles/documentation/test-api/intercepting-and-mocking-http-requests.md

```diff
@@ -0,0 +1,547 @@
+`stringifyResponseBody` | Boolean | Specifies whether the response body should be stored as a string or an object. | `false`
+
+```js
+import { RequestLogger } from 'testcafe';
+
+const simpleLogger = RequestLogger('http://example.com');
+const headerLogger = RequestLogger(/testcafe/, {
+    logRequestHeaders: true,
+    logResponseHeaders: true
+});
+```
+
+To enable the logger to track the requests, [attach it to a test or fixture](#attaching-hooks-to-tests-and-fixtures).
+
+By default, `RequestLogger` stores the following parameters.
+
+* The URL where the request is sent.
+* The request's HTTP method.
+* The status code received in the response.
+* The user agent that sent the request.
+* The ID of the test run that sent the request.
+
+To access data stored by the logger, use its API.
+
+### Methods
+
+Method | Return Type | Description
+------ | ----------- | -------------
+`contains(predicate)` | Promise | Returns whether the logger contains a request that matches the predicate.
+`count(predicate)`    | Promise | Returns the number of requests that match the predicate.
+`clear()`             | None    | Clears all logged requests.
+
+The `predicate` functions take a single parameter - the `Request` object.
+
+### Properties
+
+Property | Type | Description
+-------- | ---- | -----------
+`requests` | Array of Request | Returns an array of logged requests.
+
+### The Request Object
+
+This object represents a request-response pair.
+
+Property | Type | Description
+-------- | ---- | -----------
+`id`         | String | The ID of the request (generated by the logger).
+`testRunId`  | String | The ID of the test run that sent the request. Use it to identify the test in which the request originated.
+`userAgent`  | String | The user agent that sent the request.
+`request.url`    | String | The URL where the request is sent.
+`request.method`     | String | The request's HTTP method.
+`request.headers`    | Object | Request headers in the property-value form. Logged if the `logRequestHeaders` option is set to `true`.
+`request.body`    | Object &#124; String | The request body. An object or string depending on the `stringifyRequestBody` option. Logged if the `logRequestBody` option is set to `true`.
+`response.statusCode` | Number | The status code received in the response.
+`response.headers`    | Object | Response headers in the property-value form. Logged if the `logResponseHeaders` option is set to `true`.
+`response.body`    | Object &#124; String | The response body. An object or string depending on the `stringifyResponseBody` option. Logged if the `logResponseBody` option is set to `true`.
+
+```js
+import { RequestLogger } from 'testcafe';
+
+const logger = RequestLogger('http://api.example.com');
+
+fixture `test`
+    .page('http://example.com');
+
+test
+    .requestHooks(logger)
+    ('test', async t => {
+        const r = logger.requests[0];
```

Comment:
Meaningful name for this const is `const request = logger.requests[0];`. But in this case you have to write ugly code
`console.log(request.request.url); ` . Thus, an array of `requests`  contains no request but logs, otherwise e.g. `id` would be in `request.request.id`. Don't you think so? @VasilyStrelyaev @churkin @AlexanderMoskovkin Could we find more suitable name?

Reply:
I've changed the variable name to `logRecord`

Reply:
You can use all power of es6 and write
```js
const { request } = logger.requests[0];
```
`logRecord` is ok for me too.